src/components/formik-radiobutton/FormikRadiobutton.tsx

```diff
@@ -107,9 +127,18 @@ export class FormikRadiobutton extends React.Component<Props> {
     const errors = touched ? (this.props.serverErrors && this.props.serverErrors[name]) || error : null;
 
     const checked = this.props.value == value;
+    let tooltip;
+
+    if(this.props.tooltip){
```

Comment:
Ik denk dat het geen goed idee is om tooltips deel te laten uitmaken van een component. Beter in een flexboxed wrapper stoppen en de tooltip erachter steken. Wat denkt @simonpeters ?

Reply:
Ok dus voorlopig zo laten gelijk je het geimplementeerd hebt.